src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -652,17 +654,46 @@ public class SimpleEvents {
 					"\tcancel the event")
 				.since("2.7");
 		}
+
+		if (Skript.classExists("io.papermc.paper.event.player.PlayerStopUsingItemEvent")) {
+			Skript.registerEvent("Stop Using Item", SimpleEvent.class, PlayerStopUsingItemEvent.class, "[player] stop (using item|item use)")
```

Comment:
thoughts on `[player] item use end` as an option? e.g. `on item use end`

Reply:
I'm not a big fan of that, honestly. Feels very stilted.
I'm good with adding end to the existing structure, though: `[player] (stop|end) (using item|item use)`